docs/source/sample_policies.rst

```diff
@@ -467,3 +467,156 @@ This policy can be defined for any PolicyKit community (a Slack community, for e
 
     # Delete the topic
     metagov.perform_action("discourse.delete-topic", {"id": action.event_data["id"]})
+
+Vote on Adding Payment Pointers to a Web Monetization Rev Share config
+----------------------------------------------------------------------
+
+When a Discourse user adds a wallet to their profile, start a vote on whether to add the wallet to the community's `probabilistic revenue share config <https://webmonetization.org/docs/probabilistic-rev-sharing/>`_.
+This policy assumes that there is a custom `User Field <https://meta.discourse.org/t/how-to-create-and-configure-custom-user-fields/113192>`_ in Discourse in position "1" that holds an UpHold or GateHub wallet payment pointer.
+This policy also assumes that the Discourse server has the experimental `Metagov Web Monetization Discourse plugin <https://github.com/metagov/discourse-web-monetization>`_ installed, to generate revenue from forum content in the form of Web Monetization micropayments. All content generated on Discourse will be split equally between all wallets rev share config, which is stored in Metagov.
+
+**Required Metagov Plugins**: ``discourse``
+
+**Filter:**
+
+.. code-block:: python
+
+    is_user_fields_changed = action.action_codename == "metagovaction" and action.event_type == "discourse.user_fields_changed"
+    if not is_user_fields_changed:
+      return False
+
+    user = action.event_data["username"]
+    custom_wallet_field_key = "1"
+    old_wallet = action.event_data.get("old_user_fields", {}).get(custom_wallet_field_key)
+    new_wallet = action.event_data.get("user_fields", {}).get(custom_wallet_field_key)
+    if old_wallet == new_wallet:
+      debug(f"no wallet change for {user}, they must have changed another field. skipping.")
+      return False
+
+    debug(f"User {user} changed their wallet from '{old_wallet}' to '{new_wallet}'")
+    action.data.set("old_wallet", old_wallet)
+    action.data.set("new_wallet", new_wallet)
```

Comment:
@amyxzhang I follow this pattern frequently, using `action.data` to store variables that I need to access in future evaluation steps. This might become an issue when actions are evaluated against multiple policies, because there would be other policies that use the same key in the datastore. Really I only need these variables to be local to the action+policy pair, so maybe we could introduce a datastore just for that. Super low importance, I think, but just a thought